src/generators/biome/BiomeGenerator.res

```diff
@@ -0,0 +1,182 @@
+let requireImage = fileName => Generator.requireImage("./images/" ++ fileName)
+let requireTexture = fileName => Generator.requireImage("./textures/" ++ fileName)
+
+let id = "biome"
```

Comment:
Let's leave the Biome generator for another PR and delete from this PR.

Reply:
Done